chart/open-feature-operator/values.yaml

```diff
@@ -2,6 +2,11 @@
 ## @section Global
 ## @param  defaultNamespace To override the namespace use the `--namespace` flag. This default is provided to ensure that the kustomize build charts in `/templates` deploy correctly when no `namespace` is provided via the `-n` flag.
 defaultNamespace: open-feature-operator-system
+namespace:
+  name: open-feature-operator-system
+  create: true
+  labels: {}
+  annotations: {}
```

Comment:
How does this work in conjunction with the old `defaultNamespace`? What happens if you set `defaultNamespace` to something different than `namespace.name`? Should we remove `defaultNamespace`?